tests/shell_scripts/main.sh

```diff
@@ -75,3 +75,10 @@ TEST_NAME=0006.txt
 INPUT_CMDS="echo -n Hello World ! | /bin/cat"
 EXPECTED_EXIT_STATUS=0
 do_test
+
+# builtin env関数
+TEST_NAME=0007.txt
```

Comment:
数字でのナンバリングってブランチ切って作業するのに恐ろしく向いてないですね。命名しやすいものは命名しちゃったほうが良さそう。
```suggestion
TEST_NAME=buitlin_env.txt
```

Reply:
あーですね。
buitlin_env_0001でもいいから分類名はつけるようにします！